projects/go-lib/src/lib/components/go-portal/go-portal-attach-to.directive.ts

```diff
@@ -0,0 +1,25 @@
+import {Directive, Input, OnDestroy, OnInit, TemplateRef} from '@angular/core';
+import {GoPortalService} from './go-portal.service';
+
+@Directive({
+  selector: '[goPortalAttachTo]'
+})
+export class GoPortalAttachToDirective implements OnInit, OnDestroy{
+  @Input('goPortalAttachTo') targetName: string;
+
+  constructor(
+      private portalService: GoPortalService,
+      private template: TemplateRef<any>
```

Comment:
Looks like this has a `4` space indentation, should we change this to just `2` space?